src/communication/network_interface_sr.c

```diff
@@ -7347,6 +7347,12 @@ smmon_get_module_info (THREAD_ENTRY * thread_p, unsigned int rid, char *request,
 
   if (module_index == 0)
     {
+      /* If there is no modules that registered in memory monitoring manager module,
```

Comment:
```suggestion
      /* If there is no modules that registered in memory monitoring manager,
```

Reply:
MMON_MODULE_ID is already opened to the client. So, this cheking can be executed in the client-side.

Reply:
You mean that moving this checking phase to the client-side is a better approach?

Reply:
Yes, I do. Do you have any other opinion?

Reply:
I think it would be add assert for it instead of handling of this case. Because at least one moudle(heap) is added after terminating this project.

Reply:
I prevent to request to server at memmon(). Please check it.